### PR TITLE
Improve swagger descriptions.

### PR DESCRIPTION
**Pull request by DailyDreaming**
Just updates and adds swagger descriptions.  Should address `dcp-cli` issue #221.

### CODE REVIEW

File: dss-api.yml [hunk 1]
```diff
@@ -1431,6 +1431,23 @@ paths:
             type: object
             properties:
               files:
+                description: >
+                  The list of files making up the contents of the new bundle.  This is a list of dictionaries, each one
+                  describing the file, required, and including:
```
Comment (Bento007) [inline dss-api.yml]: This sentence is awkward, maybe try something like this.
"This is a list of dictionaries describing each of the files. Each dictionary includes the fields:"

File: dss-api.yml [hunk 2]
```diff
@@ -1431,6 +1431,23 @@ paths:
...
+                  - The "uuid" of a file already previously uploaded with "PUT file/{uuid}".
+                  - The "version" timestamp of the file.
+                  - The "name" of the file. This can be most anything, & is the name the file will have when downloaded.
```
[the remark this answers is not in the document]
Comment (DailyDreaming) [reply dss-api.yml]: Done.

File: dss-api.yml (hunk 1 above)
Comment (DailyDreaming) [reply dss-api.yml]: Done.

**Comment by Bento007** [comment]
@DailyDreaming remove #221 from the title before committing

**Pull request merged**